.github/workflows/main.yaml

```diff
@@ -0,0 +1,73 @@
+name: Main CI
+
+on: [push]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    strategy:
+      matrix:
+        python-version:
+          - '3.12'
+    steps:
+    - uses: actions/checkout@v4
+    - name: Use Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+        cache: 'pip'
+    - name: Install testing dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements-test.txt
+    - name: Lint
+      # FIXME
+      continue-on-error: true
+      run: |
+        flake8 lib/pyld tests --count --show-source --statistics
+      env:
+        LOADER: ${{ matrix.loader }}
+  test:
+    runs-on: ubuntu-latest
+    needs: [lint]
+    timeout-minutes: 10
+    strategy:
+      matrix:
+        python-version:
+          - '3.8'
```

Comment:
I :clap: applaud the decision to deprecate currently unsupported Python versions, which are < 3.8.